src/vivarium_public_health/utilities.py

```diff
@@ -112,3 +112,6 @@ def get_lookup_columns(
         necessary_columns.remove("year")
 
     return list(necessary_columns)
+
+
+NUM_RR_EXPOSURE_VALUES = 1000
```

Comment:
I stand by this should be configurable if it is only used for this new class but could have this as a default but this should be in a constants file and not utilities.

Reply:
I wasn't sure where to put this but I saw other constants get imported from this file so I put it here. But it's getting removed anyway haha